SCClassLibrary/Common/Streams/Pspawner.sc

```diff
@@ -1,8 +1,8 @@
 Spawner : Pattern {
-	var <>genStream;
-	var <>priorityQ;
-	var <>now;
-	var <>event;
+	var genStream;
+	var priorityQ;
+	var now;
+	var event;
```

Comment:
I did not see any cases of these variables used outside of this class. They may as well be private.